analytics/.eslintrc

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Great to see the usage of a linting utility, awesome! We shall make great usage of it when enforcing our coding guidelines that we shall agree on as a team.